device/bgpv4.yaml

```diff
@@ -8,59 +8,134 @@ components:
   schemas:
     Device.Bgpv4:
       description: >-
-        Emulated bgpv4 protocol
+        Emulated BGPv4 router and routes
       type: object
       required: [name]
       properties:
         name: '../common/common.yaml#/components/x-inline/UniqueName'
         as_number_2_byte:
+          description: >-
+            Autonomous system (AS) number of 2 byte
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         dut_as_number_2_byte:
+          description: >-
+            Autonomous system (AS) number of the BGP peer router (DUT)
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         as_number_4_byte:
+          description: >-
+            Autonomous system (AS) number of 4 byte
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         as_number_set_mode:
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         as_type:
-          description: >- 
-            The type of BGP autonomous system. 
-            External BGP (EBGP) is used for BGP links between two or more 
+          description: >-
+            The type of BGP autonomous system.
+            External BGP (EBGP) is used for BGP links between two or more
             autonomous systems.
             Internal BGP (IBGP) is used within a single autonomous system.
           type: string
           enum: [IBGP, EBGP]
         hold_time_interval:
+          description: >-
+            Number of seconds the sender proposes for the value of the Hold Timer
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         keep_alive_interval:
+          description: >-
+            Number of seconds between transmissions of Keep Alive messages by router
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         graceful_restart:
-          $ref: './patterns.yaml#/components/schemas/Device.Pattern'
+          description: >-
+            Graceful Restart allows the BGP forwarding state to be preserved during a BGP restart
+          type: boolean
         authentication:
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         ttl:
+          description: >-
+            The limited number of iterations that a unit of data can experience before the data is discarded.
```

Comment:
can experience what? before the data is discarded?

Reply:
Current available test case do not require authentication and ttl. I have remove those. I will add with proper description when those will require.